realm/realm-annotations-processor/src/test/resources/io/realm/AllTypesRealmProxy.java

```diff
@@ -900,9 +899,9 @@ public static long insert(Realm realm, some.test.AllTypes object, Map<RealmModel
         if (realmGet$columnBinary != null) {
             Table.nativeSetByteArray(tableNativePtr, columnInfo.columnBinaryIndex, rowIndex, realmGet$columnBinary, false);
         }
-        io.realm.MutableRealmInteger realmGet$columnMutableRealmInteger = ((AllTypesRealmProxyInterface) object).realmGet$columnMutableRealmInteger();
-        if ((realmGet$columnMutableRealmInteger != null) && (realmGet$columnMutableRealmInteger.get() != null)) {
-            Table.nativeSetLong(tableNativePtr, columnInfo.columnMutableRealmIntegerIndex, rowIndex, realmGet$columnMutableRealmInteger.get().longValue(), false);
+        Long realmGet$columnMutableRealmInteger = ((AllTypesRealmProxyInterface) object).realmGet$columnMutableRealmInteger().get();
+        if (realmGet$columnMutableRealmInteger != null) {
```

Comment:
I'm not sure about that.  This is exactly the same semantics as a "Long".  I need to look into this.  Will add "FIXME"